apps/test/unit/templates/sectionProgressV2/ProgressFeedbackBannerTest.jsx

```diff
@@ -0,0 +1,144 @@
+import {fireEvent, render, screen, waitFor} from '@testing-library/react';
+import React from 'react';
+import sinon from 'sinon';
+
+import {UnconnectedProgressFeedbackBanner} from '@cdo/apps/templates/sectionProgressV2/ProgressFeedbackBanner';
+import i18n from '@cdo/locale';
+
+import {expect} from '../../../util/reconfiguredChai';
+
+describe('UnconnectedProgressFeedbackBanner', () => {
+  const fakeFetch = sinon.spy();
+  const fakeCreate = sinon.spy();
+  const defaultProps = {
+    currentUser: {isAdmin: false},
+    canShow: false,
+    isLoading: false,
+    progressV2Feedback: {empty: true},
+    fetchProgressV2Feedback: fakeFetch,
+    createProgressV2Feedback: fakeCreate,
+    errorWhenCreatingOrLoading: null,
+  };
+
+  afterEach(() => {
+    fakeCreate.resetHistory();
+  });
+
+  it('renders correctly with initial state', () => {
+    render(<UnconnectedProgressFeedbackBanner {...defaultProps} />);
+    const questionText = screen.queryByText(
+      i18n.progressV2_feedback_question()
+    );
+    const shareMoreText = screen.queryByText(
+      i18n.progressV2_feedback_shareMore()
+    );
+    expect(questionText).to.not.exist;
+    expect(shareMoreText).to.not.exist;
+  });
+
+  it('renders correctly when user has not answered survey question', () => {
+    render(
+      <UnconnectedProgressFeedbackBanner {...defaultProps} canShow={true} />
+    );
+    expect(screen.getByText(i18n.progressV2_feedback_question())).to.be.visible;
+    expect(screen.getByTitle(i18n.progressV2_feedback_thumbsUp())).to.be
+      .visible;
+    expect(screen.getByTitle(i18n.progressV2_feedback_thumbsDown())).to.be
+      .visible;
+  });
+
+  it('renders correctly when the data is loading', () => {
+    render(
+      <UnconnectedProgressFeedbackBanner
+        {...defaultProps}
+        canShow={true}
+        isLoading={true}
+      />
+    );
+    const questionText = screen.queryByText(
+      i18n.progressV2_feedback_question()
+    );
+    const shareMoreText = screen.queryByText(
+      i18n.progressV2_feedback_shareMore()
+    );
+    expect(questionText).to.not.exist;
+    expect(shareMoreText).to.not.exist;
+  });
+
+  it('clicking thumbs up attempts to send feedback and asks for more detailed feedback', async () => {
+    render(
+      <UnconnectedProgressFeedbackBanner {...defaultProps} canShow={true} />
```

Comment:
optional nit: seems like we are overriding canShow more often than not. should we make that the default and override false otherwise?

Reply:
I thought of that too... I also think of default as the "normal" status.  Most of the time this doesn't show, but in most of the tests we are testing what IS shown.  BUT, I get that we would have less repetition if we changed the default.   I'll make the change!